docs/container.md

```diff
@@ -1,8 +1,16 @@
 # Running in a container
 
+There's an easy method and an *easier* method for running `PEPPRO` in a container. Both require you are set up to use *either* docker or singularity.
+
+## The *easier* method
+
+The absolute easiest method to utilize containers is to install one more package and use [`bulker`](https://bulker.databio.org/en/latest/peppro/), our multi-container environment manager.  Check out the [step-by-step guide to run `PEPPRO` with `bulker`](https://bulker.databio.org/en/latest/peppro/).
+
+## The easy method
+
```

Comment:
I think this is just too much complexity. Do you really want to maintain a monolithic container for this? I'd just eliminate the 'easy method' and supplant with bulker. -- I'm not willing to maintain all these different ways to do this.

If you really want to maintain both ways... then OK, that's alright with me.

Reply:
Gotcha.  I guess I was just thinking both currently existed so there was nothing saying they had to both *continue* to exist.  But since they both did it wasn't hurting anything to report both options.  In that perspective is it okay to still mention both approaches?

Reply:
But with the idea that in the not so distant future it will likely reduce to just bulker? Or do you still think it should just be bulker?

In that context, should I create a separate PEPPRO page based guide or just continue to direct people to the bulker PEPPRO guide?

Reply:
whatever you want to do on both counts, I guess.

I would probably: 
- only list bulker, for simplicity (this is kind of confusing -- the 'easy' way and the 'easier' way, etc).
- write a real tutorial using bulker for peppro, as we talked about before... the bulker one is kind of a dummy one without real data, etc.

then again, I think we have a habit of writing too much documentation that gets unmaintainable. So, maybe invoking thoreau's 'simplify simplify simplify' here would be good, and just skip the container, and skip another example.

Reply:
but then maybe we should at some point update that step-by-step guide to be on real data.